rust/azure_iot_operations_services/src/azure_device_registry/models/asset.rs

```diff
@@ -420,6 +588,16 @@ pub enum ActionType {
     Write,
 }
 
+/// Represents the MQTT Quality of Service level.
+/// Currently does not include Quality of Service 2.
+#[derive(Clone, Debug)]
+pub enum Qos {
```

Comment:
```suggestion
pub enum QoS {
```
For our enum, can we use the more correct capitalization scheme?

Reply:
Sure